Documentation update after removal of `kyma CLI` invocations (#575)
kyma-project/btp-manager: docs/contributor/03-10-release.md

   iii. click  **Run workflow** on the right  
   iv. provide a version, for example, 1.2.0  
   v. choose real or dummy credentials for Service Manager  
   vi. choose whether to bump or not to bump the security scanner config  
   vii. choose whether you want to publish the release
2. The GitHub action, defined in the [`create-release.yaml`](/.github/workflows/create-release.yaml) file, validates the release by checking if the GitHub tag already exists, if there are any old Docker images for that GitHub tag, and if merged PRs that will be part of this release are labeled correctly.
3. If you chose in step 1.vi to bump the security scanner config, the GitHub action creates a PR with a new security scanner config that includes the new GitHub tag version.
4. A code owner approves the PR. 

> 5. The GitHub action creates a GitHub tag and draft release with the provided name, creates release assets and uploads these.

Reviewer: ```suggestion
5. The GitHub action creates a GitHub tag and draft release with the provided name; then, it creates release assets and uploads them.
```

   iv. provide a version, for example, 1.2.0  
   v. choose real or dummy credentials for Service Manager  

>    vi. choose whether to bump or not to bump the security scanner config

Reviewer: ```suggestion
   vi. choose whether to bump or not to bump the security scanner config  
```

3. If you chose in step 1.vi to bump the security scanner config, the GitHub action creates a PR with a new security scanner config that includes the new GitHub tag version.
4. A code owner approves the PR. 
5. The GitHub action creates a GitHub tag and draft release with the provided name, creates release assets and uploads these.
6. The GitHub action asynchronously initiates unit tests and an await for Prow Jobs status.
7. The tag creation triggers Prow Job, `post-btp-manager-build`, defined in [`btp-manager-build.yaml`](https://github.com/kyma-project/test-infra/blob/main/prow/jobs/kyma-project/btp-manager/btp-manager-build.yaml).
8. `post-btp-manager-build` builds a Docker image tagged with the release name.
10. The GitHub action asynchronously initiates stress tests jobs and E2E tests jobs upon Prow job success status. E2E upgrade tests run only with real credentials for Service Manager.
12. The GitHub action runs E2E tests in parallel on the k3s clusters for the most recent k3s versions and with the specified credentials. The number of the most recent k3s versions to be used is defined in the **vars.LAST_K3S_VERSIONS** GitHub variable. 

> 13. If the unit tests, stress tests, and E2E tests are completed successfully (and you did not choose to not publish in 1.vii), the GitHub action publishes the release.

Reviewer: ```suggestion
13. If the unit tests, stress tests, and E2E tests are completed successfully and you have chosen to publish in 1.vii, the GitHub action publishes the release.
```
